packages/home-connector/src/adapters/access-networks-unleashed/repository.node.test.ts

```
@@ -33,13 +33,18 @@ function createConfigBase(dbPath: string) {
 		accessNetworksUnleashedScanCidrs: ['192.168.1.10/32'],
 		accessNetworksUnleashedAllowInsecureTls: true,
 		accessNetworksUnleashedRequestTimeoutMs: 8_000,
+		islandRouterApiBaseUrl: 'https://my.islandrouter.com',
```

Biome (2.4.14)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)